src/kernels/MIOpenTensorKernels.cl

```diff
@@ -105,11 +105,15 @@ __kernel void OpTensorFwdBias(global MIOPEN_TYPE* a,
         // number of workgroups = c_c (b_c)
         while(lid < work_per_wg)
         {
-            int o_hw     = incr_wg == 0 ? (lid % (work_per_wg / c_n)) : lid;
-            int o_n      = incr_wg == 0 ? (lid / (work_per_wg / c_n)) : (gid / b_c);
-            int index    = o_n * c_nstride + o_c * c_cstride + o_hw;
-            c_off[index] = MIOPEN_TENSOR_OP(a_off[index] * alpha0, operand) + beta * c_off[index];
+            int o_hw  = incr_wg == 0 ? (lid % (work_per_wg / c_n)) : lid;
+            int o_n   = incr_wg == 0 ? (lid / (work_per_wg / c_n)) : (gid / b_c);
+            int index = o_n * c_nstride + o_c * c_cstride + o_hw;
 
+#if MIO_BETA_ZERO == 1
```

Comment:
[Recommendation] I think that `MIO_IGNORE_INPUT_TENSOR_C` is a better name for this macro.

Reply:
I want to convey this change is made for BETA zero case.  Is it OK for you to keep it as "MIO_BETA_ZERO"?

Reply:
Yes, but in this case we need some textual description of the semantics of this macro to be added in comments. It can be borrowed from https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1623#discussion_r926106407.

Reply:
yes, I will add some textual description of the semantics of this macro in the code.